### PR TITLE
Update Getting Started to add Discord.JS

### PR DESCRIPTION
**Pull request by FayneAldan**
### Description of the PR

Added Discord.JS to the `npm install` command on the Getting Started page

### Changes Proposed in this Pull Request (List new items in CHANGELOG.MD)

N/A

### Semver Classification

- [X] This PR only includes documentation or non-code changes.
- [ ] This PR fixes a bug and does not change the (intended) framework interface.
- [ ] This PR adds methods or properties to the framework interface.
- [ ] This PR removes or renames methods or properties in the framework interface.

### CODE REVIEW

**Comment by Tylertron1998** [comment]
"About:
Klasa is an OOP discord.js bot framework which aims to be the most feature complete, while feeling like a consistant extension of discord.js.

Originally based on Komada, Klasa has become a ship of Theseus and so much more. You can only vaguely see similarities between the two frameworks anymore..." If you don't know you are meant to install discord.js, then you are an idiot.

File: guides/Getting Started/GettingStarted.md [hunk 1]
```diff
@@ -5,13 +5,13 @@ Time to take the plunge! Klasa is on NPM and can be easily installed.
 > I assume you know how to open a command prompt in a folder where you want to install this. Please don't prove me wrong.
 
 ```sh
-npm install --save klasa
+npm install --save discordjs/discord.js klasa
```
Comment (Tylertron1998) [inline guides/Getting Started/GettingStarted.md]: This is wrong, stable uses d.js stable.

**Comment by FayneAldan** [comment]
It was already discussed in the Discord server that master Discord.js should be used

**Comment by Tylertron1998** [comment]
So it seems. From the package.json, it appears that stable uses master d.js.

My apologise. 

My point still stands, however, you should KNOW that you need to install d.js. Should we tell people they need to install node too? And have a system to run it on? All of these things you should know, if not, you probably shouldn't be developing a bot.

**Comment by FayneAldan** [comment]
Yes, you SHOULD know that Discord.JS is needed, but nothing ever said that **master** Discord.JS is needed.

**Comment by tech6hutch** [comment]
"My point still stands, however, you should KNOW that you need to install d.js."
Not necessarily. It's a peer dep, but if it was a normal dep, it would be installed automatically.

**Pull request merged**